pkg/controller/operator/seed/resources/kubermatic/kubermatic.go

```diff
@@ -84,10 +88,18 @@ func BackupContainersConfigMapCreator(cfg *operatorv1alpha1.KubermaticConfigurat
 
 			c.Data[storeContainerKey] = cfg.Spec.SeedController.BackupStoreContainer
 			c.Data[cleanupContainerKey] = cfg.Spec.SeedController.BackupCleanupContainer
-			if cfg.Spec.SeedController.BackupRestore.Enabled {
-				c.Data[deleteContainerKey] = cfg.Spec.SeedController.BackupDeleteContainer
+
+			if cfg.Spec.SeedController.BackupStoreContainer == strings.TrimSpace(common.DefaultBackupStoreContainer) &&
+				seed.Spec.BackupRestore != nil {
```

Comment:
Does `BackupRestore` still have an `Enabled` field? should we use it here?

Reply:
No the api doesn't have that nor the seed spec. Adding this field means adding the UI and the API in addition the seed spec. TBH from my side I guess it is not intuitive for the customers to have a hint that the new backup and restore is disabled just because the fields are empty, their a better UX for this. @lsviben @floreks WDYT ?

Reply:
Anyway maybe for now we leave it as it is